openpype/settings/__init__.py

```diff
@@ -23,7 +23,7 @@
     get_project_settings,
     get_current_project_settings,
     get_anatomy_settings,
-    get_local_settings,
+    get_local_settings
```

Comment:
The commas are there so we don't have to commit 2 lines when we add new import here... (everywhere)
```suggestion
    get_local_settings,
```